src/omero/plugins/import.py

```diff
@@ -583,7 +583,11 @@ def bulk_import(self, command_args, xargs):
                             # FIXME: this assumes 'omero'
                             print(sys.argv[0], "import", rv, file=o)
                 else:
-                    self.do_import(command_args, xargs)
+                    if incr == 0:
```

Comment:
```suggestion
                    if incr == 1:
```
See the start of the loop:
```
            for cont in self.parse_bulk(bulk, command_args):
                incr += 1
```